cmd/nvidia-dra-plugin/nvlib.go

```diff
@@ -199,6 +199,10 @@ func (l deviceLib) enumerateImexChannels(config *Config) (AllocatableDevices, er
 	return devices, nil
 }
 
+func getPciAddressFromNvmlPciInfo(info nvml.PciInfo) string {
+	return fmt.Sprintf("%04x:%02x:%02x.0", info.Domain, info.Bus, info.Device)
```

Comment:
How is this different from https://github.com/NVIDIA/go-nvlib/blame/bf0cbc7fef6c51b968da7abd1e8788f0647b2266/pkg/nvlib/device/device.go#L147

Reply:
It should be the same if its returning the address in the format `0000:0a:00.0`. Wasn't aware of that implementation. I'll switch to that